api/src/main/java/org/pmiops/workbench/api/CloudTaskInitialCreditsExhaustionController.java

```diff
@@ -110,11 +98,6 @@ private void handleExhaustedUsers(Set<DbUser> newlyExhaustedUsers) {
           logger.info(
               "handleExhaustedUsers: handling user with exhausted credits {}", user.getUsername());
           initialCreditsService.updateInitialCreditsExhaustion(user, true);
-          // delete apps and runtimes
-          deleteAppsAndRuntimesInInitialCreditsWorkspaces(user);
-          // Unlink billing account in VWB
-          // This is done to stop extra charges from being incurred
-          vwbUserService.unlinkBillingAccountForUserPod(user);
```

Comment:
@tarekmamdouh, I had moved the deleteAppsAndRuntimesInInitialCreditsWorkspaces logic inside of initialCreditsService.updateInitialCreditsExhaustion to consolidate the initial credit cleanup code.

It seems like vwbUserService.unlinkBillingAccountForUserPod(user) is similar sort of cleanup code, so I moved it into the same place. Please let me know what you think.

Reply:
this looks very good, thank you Eric